### ISSUE TITLE
Peer review for hw05-schignel

### ISSUE THREAD

**Issue by ziqiangt** [STAT545-UBC-students/hw05-schignel#1]
|    **Topic**    | **Excellent** |**Satisfactory** |**Needs Work** |
|----------------|---------------|------------------|------------------|
| Coding style  |   :thumbsup:  |                        |                           |
| Coding strategy      | :thumbsup:   |                          |                           |
| Presentation: graphs |  |          :thumbsup:                 |                           |
| Presentation: tables |  :thumbsup: |                         |                          |
| Achievement, activity |    |            :thumbsup:             |                           |
| Ease of access | :thumbsup:  |                       |                       |

Remarks:
schignel has answered the basic questions and has gone beyond what was expected and required. Here are some comments that might interest you. 
1. In task "File I/O", it seems that you did not try "saveRDS()/readRDS(), dput()/dget()". It would be much better if you could find the difference between those functions.
2. In task "Writing figures to file", it would be nice if you could put the figure in the md file. You could try **![figure](relative location)** in the markdown file.
3. The plotly figure I cannot see either. Maybe this link **[Getting Started with Plotly for R](https://plot.ly/r/getting-started/)** would do what you expected. 

In conclusion, it is a good sample answer and I learned a lot from you.
